api/parts/mgmt/apps.js

```diff
@@ -241,15 +241,27 @@ appsApi.createApp = function(params) {
     newApp.edited_at = newApp.created_at;
     newApp.owner = params.member._id + "";
     newApp.seq = 0;
+    let seed = '';
+    try {
+        seed = await new Promise((resolve, reject) => {
+            crypto.randomBytes(256, (err, buf) => {
+                if (err) {
+                    reject(err);
+                }
+                resolve(buf.toString('hex'));
+            });
+        });
+    }
+    catch (e) {
+        console.log(e);
+        return common.returnMessage(params, 500, "Error creating App: " + e);
```

Comment:
Can we maybe fallback to some other seed method here? Like OS generated random numbers?